hieradata/nodes/pandemic.yaml

```diff
@@ -2,6 +2,18 @@ classes:
     - ocf_kvm
     - ocf_mesos::slave
     - ocf_mesos::secrets
+    - ocf_ceph::mon
+    - ocf_ceph::osd
+
+ocf_ceph::osd::disks:
+    - /dev/sda
```

Comment:
I kind of wish there was a better way to specify this

Reply:
You could probably do something like /dev/sda - /dev/sdh if somebody wanted to write a parser for that, I don't think it's worth the extra complexity though